requirements/requirements.txt

```diff
@@ -1,11 +1,9 @@
 # version fix
-PyYAML~=5.4
-neon-mq-connector~=0.3.0
```

Comment:
@NeonDaniel Removal of `PyYAML` and `neon-mq-connector` deps is ok
The question is only do we assume that people who install it will use `pip>20.4` that has dependency resolver?
Or else it will cause error

Reply:
It depends on the other packages in the environment; I don't think this package uses either of those directly, which is why I removed them (also unit tests passed with these removed, but not with `ovos-lingua-franca` removed).

I think it is up to pip or the installing user to deal with dependencies; specifying these here could break packages that move to newer versions